mne/stats/regression.py

```diff
@@ -134,3 +136,193 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
+                          covariates=None, reject=None, tstep=1.,
+                          decim=1, picks=None, solver='pinv'):
```

Comment:
wrt. the string formatting thing, note I copied that one verbatim from some other example. I'll fix, but there's some other example doing precisely that.